src/org.ts

```diff
@@ -193,6 +193,25 @@ export class Org extends AsyncCreatable<Org.Options> {
     });
   }
 
+  /**
+   *
+   * @param sandboxReq SandboxRequest options to create the sandbox with
+   * @param sandboxName
+   * @param options Wait: The amount of time to wait before timing out, Interval: The time interval between polling
```

Comment:
You might want to augment this a bit by adding the return object jsDoc and defining what the default options are.  Maybe even create a type/interface for the options?

Reply:
@WillieRuemmele you used the same options and return values for `createSandbox` you think is worth making an interface for just for the two options ? what are their default values ? If you think is worth making an interface we should add it to both `cloneSandbox` & `createSandbox` class members, let me know what you think.